WTWSMS/decisions/VIET_roleplay_decisions.txt

```diff
@@ -12,7 +12,6 @@ decisions = {
 		potential = {
 			is_ruler = yes
 			is_landed = yes
-			ai = no
```

Comment:
Because if all baronets do this the mod crumbles. Not for everyone, but we have to have some complacency for those who may have slower computers. Having it for the AI wouldn't add that much.